src/core/utils/misc.cpp

```diff
@@ -553,7 +553,33 @@ void Utils::Misc::openFolderSelect(const QString& absolutePath)
     if (QFileInfo(path).exists()) {
         // Syntax is: explorer /select, "C:\Folder1\Folder2\file_to_select"
         // Dir separators MUST be win-style slashes
-        QProcess::startDetached("explorer.exe", QStringList() << "/select," << Utils::Fs::toNativePath(absolutePath));
+
+        // QProcess::startDetached() has an obscure bug. If the path has
+        // no spaces and a comma(and maybe other special characters) it doesn't
```

Comment:
I'd stress the fact that commas are command line switches separator and that must always be wrapped when part of a path to a file. Something like:

explorer.exe uses commas as command line switch separator, so if a path contains a comma, this must be escaped with double quotes. QProcess::startDetached() wraps arguments between quotes only if they contain white spaces, so if a path contains a comma, but no white-spaces, explorer.exe will not open the correct path.